### PR TITLE
😭Improved work command 

### PR DESCRIPTION
**Pull request by Znunu**

### CODE REVIEW

**Comment by AstolfoDev** [comment]
bad! Y no emotes

**Comment by Znunu** [comment]
Fixed

**Comment by AstolfoDev** [comment]
10-4

File: src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/economy/Work.java [hunk 1]
```diff
@@ -40,18 +40,35 @@ public Work(EventWaiter waiter) {
     }
 
     @Override
-    protected void execute(CommandEvent e) throws NumberFormatException {
```
Comment (AstolfoDev) [inline src/main/java/tech/Astolfo/AstolfoCaffeine/main/cmd/economy/Work.java:43]: i kinda needed that lmao

**Review by AstolfoDev — COMMENTED** [review]
ight

**Pull request merged**